src/widgets/notes/encounter.resource.tsx

```diff
@@ -54,3 +56,30 @@ export function fetchEncounterByUuid(encounterUuid): Observable<any> {
     map(({ data }) => data)
   );
 }
+
+function formatNotes(notes: Array<RESTPatientNote>): Array<PatientNote> {
+  let formattedNotes: Array<PatientNote> = [];
+  notes.forEach((note: RESTPatientNote) => {
+    formattedNotes.push(mapNoteProperties(note));
+  });
+  return formattedNotes;
+}
+
+function mapNoteProperties(note: RESTPatientNote): PatientNote {
+  const formattedNote: PatientNote = {
+    id: note.uuid,
+    encounterDate: note.encounterDatetime.slice(0, 19), // TODO: FIX THIS BAD HACK

```

Comment:
Why 19? (4+1+2+1+2=10) + 1 + (2+1+2+1+2=8), like `YYYY-MM-DDTHH:MM:SS`? Why then stop? What's afterwards? If its ISO its the offset (e.g., `Z` for UTC, otherwise some offset relative to UTC). That would be relevant information for parsing / showing it.

Reply:
Yes, it's an ISO date string. Fixing this.